inc/Analytics/EnvironmentConfig.php

```diff
@@ -0,0 +1,81 @@
+<?php declare(strict_types = 1);
+
+namespace RemoteDataBlocks\Analytics;
+
+defined( 'ABSPATH' ) || exit();
+
+use Automattic\VIP\Telemetry\Tracks;
+use function Automattic\VIP\Telemetry\Tracks\get_tracks_core_properties;
+
+/**
+ * Class for environment configuration.
+ *
+ * This class abstracts WordPress-specific functions for easy mocking.
+ */
+class EnvironmentConfig {
+	/**
+	 * Tracks analytics core properties.
+	 * 
+	 * This is set by the Tracks library available in MU Plugins.
+	 *
+	 * @var array<string, mixed>
+	 */
+	private array $tracks_core_props = [];
+
+	public function __construct() {
+		if ( function_exists( 'Automattic\VIP\Telemetry\Tracks\get_tracks_core_properties' ) ) {
```

Comment:
Does this work? I would expect that backslashes need to be escaped
```suggestion
		if ( function_exists( 'Automattic\\VIP\\Telemetry\\Tracks\\get_tracks_core_properties' ) ) {
```

Reply:
Yes it works, seems like we don't need escaping in this case (using same syntax in MU Plugins also).